docs/product/base/typography.html

```diff
@@ -50,7 +50,7 @@
 <a class="td-none">…</a>
 {% endhighlight %}
         <div class="stacks-preview--example">
-            <div class="d-flex fd-column gs16 gsy">
+            <div class="d-flex fd-column gy16">
```

Comment:
### Nitpick

This page contains many flexbox elements comprised solely of `<p>` children. For this page specifically, we could probably remove the `d-flex` and gap classes altogether and have a more reasonable layout. It's not 100% necessary, but I'll leave it up to you @giamir.

Reply:
👍